### PR TITLE
Links to project page

### PR DESCRIPTION
**Pull request by Prevole**

### CODE REVIEW

**Comment by Prevole** [comment]
Ready for review

File: app/assets/javascripts/modules/reports.js [hunk 1]
```diff
@@ -202,7 +203,8 @@ angular.module('probedock.reports', [ 'ngSanitize', 'probedock.api', 'probedock.
     };
   })
 
-  .controller('ReportDetailsCtrl', function(api, $scope) {
+  .controller('ReportDetailsCtrl', function(api, orgs, $scope) {
+    orgs.forwardData($scope);
```
Comment (AlphaHydrae) [inline app/assets/javascripts/modules/reports.js]: I assume this is superfluous now that the organization is passed to the directive as a scope parameter.

File: app/views/templates/report-details.html.slim [hunk 2]
```diff
@@ -5,7 +5,9 @@
   .project-version-labels.clearfix
     .label.label-primary ng-if='!report' Loading...
     span ng-repeat='project in report.projects'
-      span.label.label-primary ng-repeat='version in report.projectVersions | filter: {projectId: project.id}' {{ project | projectName }} {{ version.name }}
+      span.label.label-primary ng-repeat='version in report.projectVersions | filter: {projectId: project.id}'
+        a.label-link href='#' ui-sref='org.projects.show({ orgName: organization.name, projectName: project.name })' {{ project | projectName }}
+        |  {{ version.name }}
```
Comment (AlphaHydrae) [inline app/views/templates/report-details.html.slim]: I would put the link around the version name as well. We might later link to the project page with some version-specific widgets pre-configured. But for the foreseeable future, we're not going to link to a page showing only information about the version and not the projet.

File: app/views/templates/reports-latest.html.slim [hunk 3]
```diff
@@ -29,11 +29,13 @@ div ng-controller='LatestReportsCtrl'
         td.date.hidden-xs.hidden-sm.hidden-md {{ report.createdAt | amDateFormat:'llll' }}
         td.details.hidden-xs
           span ng-repeat='project in report.projects'
-            span.label.label-primary ng-repeat='version in report.projectVersions | filter: {projectId: project.id}' {{ project.name }} {{ version.name }}
-            |  
+            span.label.label-primary ng-repeat='version in report.projectVersions | filter: {projectId: project.id}'
+              a.label-link href='#' ui-sref='org.projects.show({ orgName: currentOrganization.name, projectName: project.name })' {{ project.name }}
+              |  {{ version.name }}
```
Comment (AlphaHydrae) [inline app/views/templates/reports-latest.html.slim]: Same (version name in link).

File: app/views/templates/test.html.slim [hunk 4]
```diff
@@ -15,7 +15,7 @@
           div ng-if='test'
             .labels.clearfix
               .label.label-primary
-                a href='#' ui-sref='org.projects.show({ projectName: test.project.name })' {{ test.project | projectName }}
+                a.label-link href='#' ui-sref='org.projects.show({ orgName: currentOrganization.name, projectName: test.project.name })' {{ test.project | projectName }}
                 |  {{ test.projectVersion }}
```
Comment (AlphaHydrae) [inline app/views/templates/test.html.slim]: Same (version name in link).

File: app/assets/javascripts/modules/reports.js (hunk 1 above)
Comment (Prevole) [inline app/assets/javascripts/modules/reports.js]: Correct, fixed.

File: app/views/templates/report-details.html.slim (hunk 2 above)
Comment (Prevole) [inline app/views/templates/report-details.html.slim]: Fixed

File: app/views/templates/reports-latest.html.slim (hunk 3 above)
Comment (Prevole) [inline app/views/templates/reports-latest.html.slim]: Fixed

File: app/views/templates/test.html.slim (hunk 4 above)
Comment (Prevole) [inline app/views/templates/test.html.slim]: Fixed

**Pull request merged**